queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),

```

Comment:
I'm a bit confused here. Why does the default queue have size 10 (presumably empty strings), capacity 10? Also, shouldn't we have a `song` class, not just store them as `string`s?